apps/condo/pages/_app.tsx

```diff
@@ -133,8 +133,11 @@ const MenuItems: React.FC = () => {
     const isManagingCompany = get(organization, 'type', MANAGING_COMPANY_TYPE) === MANAGING_COMPANY_TYPE
     const hasAccessToTickets = get(role, 'canReadTickets', false)
     const hasAccessToIncidents = get(role, 'canReadIncidents', false)
+    const hasAccessToEmployees = get(role, 'canReadEmployees', false)
+    const hasAccessToProperties = get(role, 'canReadProperties', false)
+    const hasAccessToContacts = get(role, 'canReadContacts', false)
 
-    const { canRead: canManageNewsItems } = useNewsItemsAccess()
+    const { canRead: canReadNewsItems } = useNewsItemsAccess()
```

Comment:
```suggestion
    const { canRead: hasAccessToNewsItems } = useNewsItemsAccess()
```
this will be more consistent)